hal/src/main/native/athena/REVPDH.cpp

```diff
@@ -533,7 +533,13 @@ void HAL_GetREVPDHVersion(HAL_REVPDHHandle handle,
   version->hardwareMinor = result.hardware_minor;
   version->hardwareMajor = result.hardware_major;
   version->uniqueId = result.unique_id;
-  hpdh->versionInfo = version;
```

Comment:
This should have been `hpdh->versionInfo = *version`. I'd prefer that over the manual field copies.

Reply:
Thanks, sorry for the fumbling around.